src/ttsim/unit_converters.py

```diff
@@ -0,0 +1,976 @@
+from __future__ import annotations
+
+from typing import overload
+
+TIME_UNIT_LABELS = {
```

Comment:
```suggestion
TIME_UNIT_IDS_TO_LABELS = {
```
Always confused me what is in there, I think this is it.